src/index.html

```diff
@@ -22,6 +22,418 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="#"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item first_item">
```

Comment:
Redundant
```suggestion
          <li class="nav__item">
```

Reply:
Not fixed 
<img width="924" alt="image" src="https://github.com/user-attachments/assets/fd835f5a-0499-4605-be20-6162b09da878" />